dsl_lang/bin/main.ml

```diff
@@ -1,9 +1,9 @@
 (* open Dsl_core
 
-(* main function *)
```

Comment:
Why are we carrying this file around? Sorry if it's required by dune or something, just curious

Reply:
I don't think we need it right now (as long as we are running stuff inside utop) but once we wish to create an executable it may be helpful. At any rate I don't think this is the PR to deal with that, but we should bring it up.